data/json/monsters.json

```diff
@@ -4613,39 +4452,6 @@
     "flags":["SEES", "SMELLS", "HEARS", "VENOM", "CHITIN", "FAT"]
     },{
     "type" : "MONSTER",
-    "id" : "mon_frog",
-    "name": "bull frog",
-    "species":"AMPHIBIAN",
-    "default_faction":"vermin",
-    "symbol":"b",
-    "color":"green",
-    "size":"TINY",
-    "material":"flesh",
-    "diff":1,
-    "aggression":0,
-    "morale":0,
-    "speed":80,
-    "melee_skill":0,
-    "melee_dice":0,
-    "melee_dice_sides":0,
-    "melee_cut":0,
-    "dodge":6,
-    "armor_bash":0,
-    "armor_cut":0,
-    "hp":3,
-    "death_function":"NORMAL",
-    "special_attacks":[
-        {
-            "type" : "leap",
-            "cooldown" : 10,
-            "max_range" : 5
-        }
-    ],
-    "description":"The American bullfrog, in its natural habitat.  It feeds on insects, mice, lizards and any other living thing it can stuff down its gullet.",
-    "flags":["SEES", "SMELLS", "HEARS", "SWIMS", "HUNTS_VERMIN"],
```

Comment:
To put it another way, bullfrogs are going to suffer from all the bugs listed for vermin, and a few more specifically since it does not have the vermin flag.